modules/aws-config/modules/org-conformance-pack/README.md

```diff
@@ -0,0 +1,48 @@
+# AWS Config Conformance Pack
+
+This module deploys a
+[Conformance Pack](https://docs.aws.amazon.com/config/latest/developerguide/conformance-packs.html). A conformance pack
```

Comment:
Awkward line breaks

Reply:
this isn't a very actionable comment. Would you like everything to be on an extra long line? Or would you rather the link be separated from the text?

Reply:
@osterman let me know if there's some hard and fast rule to handling this. To be clear, it doesn't seem to affect the [rendered result](https://github.com/cloudposse/terraform-aws-components/blob/b5092ac4851a582f3c56232073370526e18bb476/modules/aws-config/modules/org-conformance-pack/README.md)

Reply:
Fair point, I don't have a rule or a way to codify it. 

For example, this would also render the same way:

```
is a collection 
of AWS Config rules and remediation 
actions 
that can be easily deployed as 
a single entity in an account
and a Region or across an organization in AWS Organizations. 
Conformance packs are created by authoring a YAML template
that contains 
the list of AWS 
Config managed or custom 
rules and remediation actions.
```

But that would be a bit awkward, IMO.

My point is, if you were not width constrained, it's probably not where the line break would naturally be placed. Use natural line breaks, not 80-characters or other arbitrary constraints. You're absolutely right, it's a minor point.

Reply:
So, I tried reflowing this and prettier snapped it back to this version. For now, I think I'll just leave it, and instead we'll need to look at the prettier config. maybe @milldr knows why this is happening?